moderator.js

```diff
@@ -0,0 +1,72 @@
+var lang = require('./languages');
+
+exports.getScriptFromGame = function (game) {
+    return _getScriptFromDeck(game.deck);
+}
+
+exports.getScriptFromDeck = function (deck) {
+    return _getScriptFromDeck(deck);
+}
+
+function _getScriptFromDeck (deck) {
+    var script = '';
+
+    // To take note calls.
+    if(deck['Lycan']) script += ' - ' + lang.english['Lycan'] + '\n'; 
+    if(deck['Tough Guy']) script += ' - ' + lang.english['Tough Guy'] + '\n'; 
+    if(deck['Hunter']) script += ' - ' + lang.english['Hunter'] + '\n'; 
+    if(deck['Fruit Brute']) script += ' - ' + lang.english['Fruit Brute'] + '\n'; 
+    if(deck['Apprentice Seer']) script += ' - ' + lang.english['Apprentice Seer'] + '\n'; 
+    if(deck['Diseased']) script += ' - ' + lang.english['Diseased'] + '\n'; 
+    if(deck['Big Bad Wolf']) script += ' - ' + lang.english['Big Bad Wolf'] + '\n'; 
+
+    // Relationships
+    if(deck['Cupid']) {
+        script += ' - ' + lang.english['Cupid'] + '\n'; 
```

Comment:
We should merge these two because the are always called just once and one after the other.